src/deployment/flavors.md

```diff
@@ -3,41 +3,159 @@ title: Creating flavors for Flutter
-  with different Firebase projects per flavor][]
-* [Flutter 1.17 — no more Flavors, no more iOS Schemas.
-  Command argument that changes everything][]
-* [Multi-environment Flutter Projects with Flavors][]
-  (including support for web, Firebase and FlutterFire CLI)
-
-The following packages are listed alphabetically:
+Imagine that you want to make both free and paid versions of your Flutter app. 
+You can use flavors to set up both app versions 
+without writing two separate apps. 
+The free version of the app has basic functionality. 
+In contrast, the paid version has basic app functionality, extra features, 
+and different styles for paid users. 
+
+Another example where you could use flavors is for feature development. 
+If you’ve built a new feature and want to try it out, 
+you could set up a flavor and deploy that feature to a different API endpoint 
+than production. Your production code remains unaffected 
+until you are ready to deploy your new feature to production.
+
+Flavors function by letting you define compile time configurations 
+and set parameters that are read at runtime depending on your app needs.
+
+This document guides you through setting up Flutter flavors for iOS and Android. 
+
+## Environment set up
+Prerequisites:
+* Xcode installed
+* Android Studio
+* An existing project 
+
+To set up flavors in iOS, you’ll define build configurations in Xcode. 
+
+## Creating flavors in iOS
+1. Open your project in Xcode.
+
+2. Select the **Runner > New Scheme** from the menu to add a new Scheme. 
+A scheme describes how Xcode runs different actions. 
+For the purposes of this guide the example _flavor_ and _scheme_ are 
+named `development`. 
+The build configurations in the development scheme 
+have the `-development` suffix. 
+
+3. Duplicate the build configurations to differentiate between the 
+default configurations that are already available and the new configurations 
+for the `development` scheme. Under the **Info** tab at the end of the 
+**Configurations** dropdown list, click the plus button and duplicate 
+each configuration name(Debug, Release, and Profile). 
+Duplicate the listed production configurations, once for each environment.   
+
+![Step 3 Xcode image](/assets/images/docs/flavors/step3-ios-build-config.png){:width="100%"}
+
+4. To match the development flavor, add `-development` 
+at the end of each new configuration name. 
+
+5. Change the `development` scheme to match the development 
+build configurations already created.
+* In the **Runner** project, click **Manage Schemes…** and a pop up window opens. 
+* Double click the development scheme. In the next step 
+(as shown in the screenshot), you’ll modify each scheme 
+to match its dev build configuration:
+
+![Step 5 Xcode image](/assets/images/docs/flavors/step5-ios-scheme.png){:width="100%"}
+
+## Using flavors in iOS
+
+Now that you’ve set up your development flavor, 
+you can add different product bundle identifiers per flavor. 
```

Comment:
Different bundle identifiers is only an _example_ of something you can do with flavors, it's not the way you use flavors per se.